specification/loadtestservice/resource-manager/Microsoft.LoadTestService/preview/2021-11-01-preview/loadtestservice.json

```diff
@@ -0,0 +1,467 @@
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "LoadTests"
+        ],
+        "description": "Create or update LoadTest resource.",
+        "operationId": "LoadTests_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LoadTestNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "LoadTestResource",
+            "description": "LoadTest resource data",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LoadTestResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/LoadTestResource"
+            }
+          },
+          "default": {
+            "description": "Resource provider error response about the failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LoadTests_CreateOrUpdate": {
+            "$ref": "./examples/LoadTests_CreateOrUpdate.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "LoadTests"
+        ],
+        "description": "Update a loadtest resource.",
+        "operationId": "LoadTests_Update",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LoadTestNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "ResourcePatchRequestBody",
+            "description": "LoadTest resource update data",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ResourcePatchRequestBody"
```

Comment:
>ResourcePatchRequestBody [](http://example.com/codeflow?start=37&length=24)

Please rename definition : "ResourcePatchRequestBody"to reflect that this is a LoadTest resourcePatch not a generic resource patch

Reply:
Done, thanks!